sdkconfig.defaults

```diff
@@ -7,14 +25,14 @@
 #
 # Serial flasher config
 #
-#CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
-#CONFIG_ESPTOOLPY_BAUD_115200B=y
+CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
+# CONFIG_ESPTOOLPY_BAUD_115200B is not set
 # CONFIG_ESPTOOLPY_BAUD_230400B is not set
-# CONFIG_ESPTOOLPY_BAUD_921600B is not set
+CONFIG_ESPTOOLPY_BAUD_921600B=y
```

Comment:
And on top of that it begins with: 'clean and update sdkconfig' :)
The update of the idf is the second part, and basicly a updated submodule
(In our IDF there is a PR which only mentions upstream pr merges ofcourse, you might be confused with that?)